bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Projects/DeleteProjectCommand.cs

```diff
@@ -72,9 +74,16 @@ public DeleteProjectCommand(IProjectRepository projectRepository, ICurrentContex
 
         if (deleteIds.Count > 0)
         {
+            var secretIds = results.SelectMany(projTuple => projTuple.Item1?.Secrets?.Select(s => s.Id) ?? Array.Empty<Guid>()).ToList();
+
+            if (secretIds != null && secretIds.Count > 0)
```

Comment:
When would this be nullable? I feel like we're checking that above with the `??`

Reply:
That's a good point. Unless results is `null` (it never should be based on line 52), `secretIds` shouldn't be `null`.

I removed the `null` check